t/plugin/authz-keycloak.t

```diff
@@ -49,12 +49,38 @@ done
 
 
 
-=== TEST 2: full schema check
+=== TEST 2: sanity (using discovery endpoint)
 --- config
     location /t {
         content_by_lua_block {
             local plugin = require("apisix.plugins.authz-keycloak")
-            local ok, err = plugin.check_schema({token_endpoint = "https://efactory-security-portal.salzburgresearch.at/",
+            local ok, err = plugin.check_schema({
+                                   discovery = "https://efactory-security-portal.salzburgresearch.at/.well-known/uma2-configuration",
```

Comment:
Aw, bummer, it seems I have used my previous dev configurations, please do change it to a generic URL if possible.

Reply:
sure